docs/develop/develop-overview.md

```diff
@@ -0,0 +1,189 @@
+
+The web application can use the Office JavaScript API to interact with content in the Office document where the add-in is running, and can also do other things that web applications typically do, like call external web services, facilitate user authentication, and more.
+
+### Defining an add-in's settings and capabilities
+
+An Office Add-in's manifest (an XML file) defines the settings and capabilities of the add-in. You can configure the manifest to specify things such as:
+
+- Metadata that describes the add-in (for example, ID, version, description, display name, default locale)
+- The Office applications where the add-in will run
+- Permissions that the add-in requires
+- How the add-in integrates with Office, including any custom UI that the add-in creates (for example, custom tabs, ribbon buttons)
+- Location of images that the add-in uses for branding and command iconography
+- Dimensions of the add-in (for example, dimensions for content add-ins, requested height for Outlook add-ins)
+- Rules that specify when the add-in activates in the context of a message or appointment (for Outlook add-ins only)
+
+For detailed information about the manifest, see [Office Add-ins XML manifest](add-in-manifests.md).
+
+### Extending the Office UI
+
+An Office Add-in can extend the Office UI by using add-in commands and HTML containers such as task panes, content add-ins, or dialog boxes.
+
+- [Add-in commands](../design/add-in-commands.md) can be used to add custom tabs, buttons, or menus to the default ribbon in Office, or to extend the default context menu that appears when users right-click text in an Office document or an object in Excel. When users select an add-in command, they initiate the task that the add-in command specifies, such as running JavaScript code, opening a task pane, or launching a dialog box.
+
+- HTML containers like [task panes](../design/task-pane-add-ins.md), [content add-ins](../design/content-add-ins.md), and [dialog boxes](../design/dialog-boxes.md) can be used to display custom UI and expose additional functionality within an Office application. The content and functionality of each task pane, content add-in, or dialog box derives from a web page that you specify. Those web pages can use the Office JavaScript API to interact with content in the Office document where the add-in is running, and can also do other things that web applications typically do, like call external web services, facilitate user authentication, and more.
+
+The following image shows an add-in command in the ribbon, a task pane to the right of the document, and a dialog box floating over the document.
+
+![An image that shows add-in commands on the ribbon, a task pane, and a dialog box in an Office document](../images/overview-with-app-interface-elements.png)
+
+For more information about extending the Office UI, see [Design Office Add-ins](../design/add-in-design.md).
+
+### Interacting with content in an Office document
```

Comment:
Interacting with the document seems more fundamental then UI parts. Can we swap their positions?